owncloudApp/build.gradle

```diff
@@ -93,7 +93,6 @@ dependencies {
 
     debugImplementation "androidx.fragment:fragment-testing:1.4.1"
     debugImplementation 'com.facebook.stetho:stetho:1.6.0'
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:2.9.1'
```

Comment:
Was this intentional? 🤨 It also bothers me sometimes, but I don't know if you wanted to remove it definitely 😀